src/Pages/Projects/Projects.js

```diff
@@ -1,201 +1,75 @@
-          {projectData.map((project) => (
-            <article className="p-6 bg-white rounded-lg border border-gray-200 shadow-md dark:bg-gray-800 dark:border-gray-700">
-              <div className="flex justify-between items-center mb-5 text-gray-500">
-                {/*  */}
-                <span className="bg-primary-100 text-primary-800 text-xs font-medium inline-flex items-center px-2.5 py-0.5 rounded dark:bg-primary-200 dark:text-primary-800">
-                  <svg className="mr-1 w-3 h-3" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
-                    <path d="M2 6a2 2 0 012-2h6a2 2 0 012 2v8a2 2 0 01-2 2H4a2 2 0 01-2-2V6zM14.553 7.106A1 1 0 0014 8v4a1 1 0 00.553.894l2 1A1 1 0 0018 13V7a1 1 0 00-1.447-.894l-2 1z">
-                    </path>
-                  </svg>
-                  {project.information}
-                </span>
-                {/* maybe put something down here later? */}
-                {/* <span className="text-sm">14 days ago</span> */}
-              </div>
-              <img src={project.image} />
-              <h2 className="mb-2 pt-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white">
-                <a href="#">{project.name}</a>
-              </h2>
-              <p className="mb-5 font-light text-gray-500 dark:text-gray-400">{project.caption}</p>
-              <div className="flex grid-flow-col justify-between items-center">
-                <div className={`hidden md:flex justify-between items-center h-10 ${project.iconDivClassName}`}>
-                  {project.icons}
-                </div>
-                <a
-                  href={project.link}
-                  className="inline-flex items-center font-medium text-primary-600 dark:text-primary-500 hover:underline"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  GitHub Link
-                  <svg className="ml-2 w-4 h-4" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
-                    <path fillRule="evenodd" d="M10.293 3.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L14.586 11H3a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z" clip-rule="evenodd">
-                    </path>
-                  </svg>
-                </a>
-              </div>
-            </article>
-          ))}
-        </div>
+    <div className="dark:bg-gray-900">
+      <div className="text-center">
+        <br />
+        <p className="text-3xl lg:text-4xl tracking-tight font-extrabold text-gray-900 dark:text-white">Our Recent Projects</p>
+        <br />
+        <p className="font-light text-gray-500 sm:text-xl dark:text-gray-400 mx-8">The SCE Development Team is open to all students, no prior experience is required!</p>
+        <br />
       </div>
-    </section>
+      {projects.map((project, index) => (
+        <>
+          <ProjectCard key={index} project={project} />
```

Comment:
see link for explanation on avoiding lists and keys
https://legacy.reactjs.org/docs/lists-and-keys.html#keys

```suggestion
      {projects.map((project) => (
        <>
          <ProjectCard key={project.link} project={project} />
```

Reply:
fixed